app/gitcoinbot/README.md

```diff
@@ -0,0 +1,14 @@
+# Gitcoinbot is a Github App for Gitcoin.co
+
+## Installation Instructions
+
+The following environment variables must be set for gitcoinbot to work correctly
+
+- GITHUB_API_USER = 'gitcoinbot' ( Should correspond to [the gitcoinbot user](https://github.com/gitcoinbot))
```

Comment:
@romanjesus mind passing your local values for these settings to myself and @mbeacom on slack?

Reply:
Yup will do.